src/OSPSuite.Core/Serialization/Xml/DataRepositoryXmlSerializer.cs

```diff
@@ -25,5 +26,11 @@ private Action<IExtendedProperty> addFunction(DataRepository dataRepository)
       {
          return dataRepository.ExtendedProperties.Add;
       }
+
+      protected override void TypedDeserialize(DataRepository dataRepository, XElement outputToDeserialize, SerializationContext serializationContext)
+      {
+         base.TypedDeserialize(dataRepository, outputToDeserialize, serializationContext);
+         serializationContext.Register(dataRepository);
```

Comment:
DataRepo are not registered in the serialization context. I DO NOT KNOW WHY and this is the change that is the most impactful. But this should be ok I think